controllers/admin/AdminCmsCategoriesController.php

```diff
@@ -221,11 +221,11 @@ public function renderForm()
                     'name' => 'name',
                     'required' => true,
                     'lang' => true,
-                    'hint' => $this->l('Invalid characters:').' &lt;&gt;;=#{}'
+                    'hint' => $this->trans('Invalid characters:', array(), 'Admin.Notifications.Error').' &lt;&gt;;=#{}'
```

Comment:
Can you please change the domain to Admin.Notifications.Info?
As it's a help text it's not technically saying a error message. Thanks :)